internal/test/framework/framework.go

```diff
@@ -164,14 +177,20 @@ func (f *TestFramework) Setup(vmCount int, credentials []*types.Credentials, ski
 	return nil
 }
 
-// getKubeClient setups the kubeclient that can be used across all the test suites.
-func (f *TestFramework) getKubeClient() error {
+// getKubeConfig sets the kubeconfig in the test framework, using the KUBECONFIG environment variable
+func (f *TestFramework) getKubeconfig() error {
```

Comment:
Why are you introducing this change that introduces a subtle ordering requirement with the `get*()` functions that is not called out anywhere? I am not sure @ravisantoshgudimetla's [comment](https://github.com/openshift/windows-machine-config-bootstrapper/pull/142#discussion_r380317780) implied that you do this.

Reply:
That is how I interpreted his comment, reading it again I think I misinterpreted it